src/extensions/behavior/Selection/selection.ts

```diff
@@ -99,18 +99,16 @@ const getTopLevelNodesFromSelection = (selection: Selection, doc: Node) => {
     const nodes: {node: Node; pos: number}[] = [];
     if (selection.from !== selection.to) {
         const {from, to} = selection;
+
         doc.nodesBetween(from, to, (node, pos) => {
             const withinSelection = from <= pos && pos + node.nodeSize <= to;
-            if (
-                node &&
-                node.type.name !== 'paragraph' &&
-                !node.isText &&
-                node.type.spec.selectable &&
-                withinSelection
-            ) {
+
+            if (node && !node.isText && node.type.spec.selectable && withinSelection) {
```

Comment:
Default value of `spec.selectable` is `true`
<img width="713" alt="image" src="https://github.com/user-attachments/assets/8cf8d86f-92c6-4602-97bc-e8ad8a875d45" />

You should replace check of `node.type.spec.selectable` with `node.type.spec.selectable !== false`